site/content/en/docs/tasks/manage/productization/cert_manager.md

```diff
@@ -37,5 +37,5 @@ if one wants to use CertManager.
 
 Kueue can also support optional helm values for Cert Manager enablement.
 
-1. Disable `internalCertManager` in the kueue configuration.
+1. Disable `internalCertManagement` in the kueue configuration.
```

Comment:
Can you file a separate PR for this?

It would be worth cherry-picking this to the website separately from this PR.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/6863